app/assets/javascripts/pageflow/editor/models/theme.js

```diff
@@ -0,0 +1,13 @@
+pageflow.Theme = Backbone.Model.extend({
+  initialize: function(attributes, options) {
+    this.options = options || {};
+  },
+
+  title: function() {
+    return I18n.t('pageflow.' + this.get('name') + '_theme.name');
+  },
+
+  thumbnailUrl: function() {
+    return this.options.thumbnailUrl || this.get('preview_thumbnail_url');
```

Comment:
This option is only used by [the theme factory](https://github.com/codevise/pageflow/pull/778/files#diff-2b3aab320c738f1a3c0e0a4451449e45R16). But the factory could just pass the `preview_thumbnail_url` attribute directly. Then all code referring to `options` in this file could be removed.